pkg/apis/dynatrace/v1alpha1/oneagent_types.go

```diff
@@ -39,11 +47,43 @@ type OneAgentSpec struct {
 	DNSPolicy corev1.DNSPolicy `json:"dnsPolicy,omitempty"`
 }
 
+type OneAgentConditionType string
+
+const (
+	ApiTokenConditionType  OneAgentConditionType = "ApiToken"
+	PaaSTokenConditionType OneAgentConditionType = "PaaSToken"
+)
+
+type OneAgentCondition struct {
+	Type   OneAgentConditionType  `json:"type"`
+	Status corev1.ConditionStatus `json:"status"`
+	// Unique, this should be a short, machine understandable string that gives the reason
```

Comment:
FYI, this unrelated comment is being included on the schema.

Reply:
Removed unnecessary comment